WcaOnRails/app/models/competition.rb

```diff
@@ -640,11 +637,11 @@ def in_progress?
     !results_posted? && (start_date..end_date).cover?(Date.today)
   end
 
-  # The name `is_probably_over` is meant to be surprising.
+  # The name `probably_over` is meant to be surprising.
   # We don't actually know when competitions are over, because we don't know their schedules, nor
   # do we know their timezones.
   # See discussion here: https://github.com/thewca/worldcubeassociation.org/pull/1206/files#r98485399.
-  def is_probably_over?
+  def probably_over?
```

Comment:
Oh cool! I thought it was a ruby thing to never prefix with `is_*` because we have this suffix with `?` convention. I do think that the `is_` reads better most of the time, however, so I'm happy to revert this and turn off the cop.